org.eclipse.xtext.p2repository/pom.xml

```diff
@@ -13,6 +13,8 @@
 	<properties>
 		<root-dir>${basedir}/..</root-dir>
 		<zipFinalName>${project.artifactId}-${qualifiedVersion}</zipFinalName>
+		<!-- Used for tycho-eclipserun-plugin -->
+		<eclipse-version>2023-03</eclipse-version>
```

Comment:
is this why we need j17?

Reply:
That version is used only to collect the p2 bundles and features to use the p2 composite ant tasks through `tycho-eclipserun-plugin` (see a few lines below). I preferred to use the latest version of the platform, but it might also work with previous versions.